sandbox/modules/identity/src/main/java/org/opensearch/identity/rest/action/permission/PERMISSION_API.yaml

```diff
@@ -0,0 +1,99 @@
+openapi: 3.0.0
```

Comment:
Could you provide a little more background on how you used this file and what it helped with?  Were you able to generate stub files from it or generate this file from the APIs implementation?

Reply:
I moved this into the docs folder. It is just supposed to give context on the API. Craig had linked this specification and asked for more information on the design so he did not need to read through to understand how you would use it and so I wrote this up for him. It has changed a bit though. The most up-to-date option is #6119 I would definitely appreciate your thoughts on it if you get a chance. I know how busy you are though and @cwperks and @DarshitChanpura have given me lots of great feedback already too.